pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -393,6 +393,10 @@ func (c *awsElasticBlockStoreUnmounter) TearDown() error {
 
 // Unmounts the bind mount
 func (c *awsElasticBlockStoreUnmounter) TearDownAt(dir string) error {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
Interested in the second approach. I'll start going through the code to make sense of what needs to be done.